nltk/corpus/reader/wordnet.py

```diff
@@ -587,6 +587,7 @@ def closure(self, rel, depth=-1):
 
     from nltk.util import acyclic_depth_first as acyclic_tree
 
+    from nltk.util import unweighted_minimum_cost_spanning_tree as mst
```

Comment:
I don't think we need "cost" in the name, do we?

Reply:
I totally agree, and changed the name accordingly,